xls2xml/xls2xml/tsv2xml.py

```diff
@@ -49,7 +52,11 @@
             child_node = etree.SubElement(element_root, tags[header])
             child_node.text = str(row.get(header, ''))
     else:
-        print 'Please fix above error at file ' + tsv_filename + '!'
+        has_error = True
```

Comment:
Can we add Row where the error occured and error msg

Reply:
The library used here for reading from TSV file does not provide the row number. In its internal it skips comment rows for example and hence any row counting on our side could be inaccurate. Hence it is better not to report which row the error happens. But the error message from the validator will contain the reason why the row fails validation.